Submitted By: R. Bosch <remybosch -at- zonnet -dot- nl>
Date: 2004-02-05
Initial Package Version: 1.1.1a
Origin: openoffice-1.1.0-no-mozab.patch (checked/remade)
Description: Prevent compilation and installation of mozilla addressbook.

diff -ru oo_1.1.1a_src/connectivity/prj/build.lst oo_1.1.1a_src-new/connectivity/prj/build.lst
--- oo_1.1.1a_src/connectivity/prj/build.lst	2003-06-02 09:53:47.000000000 +0200
+++ oo_1.1.1a_src-new/connectivity/prj/build.lst	2004-02-04 16:06:56.000000000 +0100
@@ -7,8 +7,6 @@
 cn	connectivity\source\resource			nmake	-	all	cn_res NULL
 cn	connectivity\source\sdbcx				nmake	-	all	cn_sdbcx cn_types NULL
 cn	connectivity\source\drivers\ado			nmake	-	w	cn_ado cn_dbtools NULL
-cn	connectivity\source\drivers\mozab\mozillasrc		nmake	-	all	cn_mozab_mozillasrc cn_file NULL
-cn	connectivity\source\drivers\mozab	nmake	-	all	cn_mozab cn_mozab_mozillasrc cn_dbtools NULL
 cn	connectivity\source\drivers\evoab	nmake	-	all	cn_evoab cn_dbtools cn_file NULL
 cn	connectivity\source\drivers\calc		nmake	-	all	cn_calc cn_file NULL
 cn	connectivity\source\drivers\odbc		nmake	-	all	cn_odbc cn_dbtools NULL
diff -ru oo_1.1.1a_src/scp/source/office/files.scp oo_1.1.1a_src-new/scp/source/office/files.scp
--- oo_1.1.1a_src/scp/source/office/files.scp	2003-12-13 16:36:15.000000000 +0100
+++ oo_1.1.1a_src-new/scp/source/office/files.scp	2004-02-04 16:05:21.000000000 +0100
@@ -401,19 +401,6 @@
 	Styles			= (PACKED);
 End
 
-#if !defined(MACOSX) && !defined(WITHOUT_MOZILLA)
-File GID_FILE_LIB_MOZABDRV
-	TXT_FILE_BODY;
-	#ifdef UNX
-    Name            = STRING(CONCAT2(libmozabdrv2,UNXSUFFIX));
-	#else
-    Name            = "mozabdrv2.dll";
-	#endif
-    Dir             = GID_DIR_PROGRAM;
-	Styles			= (PACKED);
-End
-#endif
-
 STD_UNO_LIB_FILE( GID_FILE_LIB_DBU, dbu )
 STD_UNO_LIB_FILE( GID_FILE_LIB_DBA, dba)
 
@@ -765,20 +752,6 @@
         #endif
 End
 
-#if !defined(MACOSX) && !defined(WITHOUT_MOZILLA)
-File GID_FILE_LIB_MOZAB_2
-        TXT_FILE_BODY;
-        Styles          = (PACKED,UNO_COMPONENT);
-        RegistryID      = GID_STARREGISTRY_SERVICES_RDB;
-        Dir             = GID_DIR_PROGRAM;
-        #ifdef UNX
-        Name            = STRING(CONCAT2(libmozab2,UNXSUFFIX));
-        #else
-        Name            = "mozab2.dll";
-        #endif
-End
-#endif
-
 #ifdef UNX
 
 File GID_FILE_LIB_EVOAB_1
